src/context-parser-handlebars.js

```diff
@@ -894,6 +963,25 @@ ContextParserHandlebars.prototype.handleRawBlock = function(input, i, saveToBuff
     throw "Parsing error! Cannot encounter '}}}}' close brace of raw block.";
 };
 
+/**
+* @function ContextParserHandlebars.reset
+*
+* @description
+* All non-config internal variables are needed to reset!
+*/
+ContextParserHandlebars.prototype.reset = function() {
```

Comment:
can you call such reset() in the constructor of ContextParserHandlebars too? maintaining them in multiple places is error-prone. thanks